lib/ts/Makefile.am

```diff
@@ -22,6 +22,10 @@ library_includedir=$(includedir)/ts
 
 library_include_HEADERS = apidefs.h
 
+atscppapi_includedir=$(includedir)/atscppapi
```

Comment:
I think it would be better to just append `string_view.h` to `library_include_HEADERS`.

Reply:
But then then string_view.h will go into include/ts not include/atscppapi .  That seems wrong.

Reply:
I think overall that's the least bad choice. I think the consistency of use internally and externally out weighs the desirability of putting this in the CPPAPI area. But let me ponder that, I see your point.